docs/tutorial.md

```diff
@@ -0,0 +1,372 @@
+---
+title: Quarkus Operator Tutorial
+linkTitle: Tutorial
+weight: 30
+description: An in-depth walkthough of building and running a Quarkus-based operator.
+---
+
+**NOTE:** If your project was created with an `operator-sdk` version prior to `v1.0.0`
+please [migrate][migration-guide], or consult the [legacy docs][legacy-quickstart-doc].
+
+## Prerequisites
+
+- Go through the [installation guide][install-guide].
+- User authorized with `cluster-admin` permissions.
+- An accessible image registry for various operator images (ex. [hub.docker.com](https://hub.docker.com/signup),
+[quay.io](https://quay.io/)) and be logged in in your command line environment.
+  - `example.com` is used as the registry Docker Hub namespace in these examples.
+  Replace it with another value if using a different registry or namespace.
+  - [Authentication and certificates][image-reg-config] if the registry is private or uses a custom CA.
```

Comment:
broken link, where is image-reg-config defined.

Reply:
Removed completely as we haven't created any image out of it.